src/k8s/pkg/component/component_test.go

```diff
@@ -266,10 +274,76 @@ func TestEnableMultipleComponents(t *testing.T) {
 		g.Expect(err).ShouldNot(HaveOccurred())
 	}
 
-	components, err := mockHelmClient.List()
-	g.Expect(err).ShouldNot(HaveOccurred())
-	g.Expect(components).To(ConsistOf(
-		Component{Name: "one", Status: true},
-		Component{Name: "two", Status: true},
-	), "Expected all components to be enabled")
+	// Reenable should not error
+	for name := range mockHelmClient.components {
+		err := mockHelmClient.Enable(name, map[string]interface{}{})
+		g.Expect(err).ShouldNot(HaveOccurred())
```

Comment:
Looked up what is best practice here as I used to do:
`g.Expect(err).To(BeNil)`

Seems like this is the recommended way (btw, prefer `any` over `interface{}`):
```
g.Expect(mockHelmClient.Enable(name, map[string]any{})).Should(Succeed())
```
https://github.com/onsi/gomega/blob/master/matchers.go#L98